client/my-sites/plans-features-main/index.tsx

```diff
@@ -717,6 +719,23 @@ const PlansFeaturesMain = ( {
 		[ simplifiedFeaturesGridExperimentVariant, translate ]
 	);
 
+	const viewAllPlansButton = (
+		<div className="plans-features-main__escape-hatch">
+			<Button
+				borderless
+				onClick={ () => {
+					if ( ! isTargetedViewForSimplifiedFeaturesGridExperiment ) {
+						setSimplifiedFeaturesGridExperimentVariantOverride( 'control' );
+					}
```

Comment:
This could have been a side effect on `forceDefaultPlans` changing, no? So we could avoid the changes to the tree nodes and introduce a separate effect that we'd discard when done. Would that be cleaner?